kiwipy/rmq/threadcomms.py

```diff
@@ -253,7 +253,7 @@ def done(_):
             except concurrent.futures.CancelledError:
                 aio_future.cancel()
```

Comment:
I guess that by a similar logic `cancel()` should also be done through `call_soon_threadsafe`

Reply:
Sure, added.